extensions/kubernetes/CMakeLists.txt

```diff
@@ -14,8 +14,7 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
-
-if (NOT (CMAKE_SYSTEM_NAME STREQUAL "Linux" AND ENABLE_KUBERNETES))
+if (NOT (CMAKE_SYSTEM_NAME STREQUAL "Linux" AND (ENABLE_ALL OR ENABLE_KUBERNETES)))
```

Comment:
You added a `NOT OPENSSL_OFF` condition to this in #1583; please rebase and resolve the conflict.

Reply:
Thanks, updated in latest push.